hawkbit-repository/hawkbit-repository-jpa/src/test/java/org/eclipse/hawkbit/repository/jpa/ControllerManagementTest.java

```diff
@@ -605,6 +710,29 @@ public void findOrRegisterTargetIfItDoesNotExistDoesntRetryWhenEntityAlreadyExis
         }
     }
 
+    @Test
+    @Description("Register a controller which does not exist, if a EntityAlreadyExistsException is raised, the "
+            + "exception is rethrown and no further retries will be attempted")
+    public void findOrRegisterTargetIfItDoesNotExistWithNameDoesntRetryWhenEntityAlreadyExistsException() {
```

Comment:
remove this test, because it does not make sense due to `findOrRegisterTargetIfItDoesNotExist` overloading

Reply:
Removed.